### PR TITLE
OCM-3300: edits to main readme file

### PR DESCRIPTION
**Pull request by aldiazRH**

### CODE REVIEW

**Comment by nirarg** [comment]
@aldiazRH
I marked this as WIP, please let me know when the PR is ready for review

**Comment by aldiazRH** [comment]
> @aldiazRH I marked this as WIP, please let me know when the PR is ready for review

Thanks, @nirarg . I plan to have whatever I've completed by week's end for review. So by next week Monday it'll be ready for a first pass.

File: README.md [hunk 1]
```diff
@@ -1,42 +1,43 @@
...
 ## Notice
 
-This module is currently in its Alpha version and remains under active development; using it carries inherent risks that you should be aware of.
+This module is currently in Alpha and remains under active development. Using it carries inherent risks that you must be aware of.
 
-For the supported provider go to https://registry.terraform.io/providers/terraform-redhat/rhcs/latest
-For the supported rosa-sts module go to https://registry.terraform.io/modules/terraform-redhat/rosa-sts/aws/latest
+For information about the supported provider, go to https://registry.terraform.io/providers/terraform-redhat/rhcs/latest
+
+For information about the supported rosa-sts module, go to https://registry.terraform.io/modules/terraform-redhat/rosa-sts/aws/latest
```
Comment (nirarg) [inline README.md:11]: Please remove this section.
As once this PR is merge, this is not true anymore :-)

File: README.md [hunk 2]
```diff
@@ -1,42 +1,35 @@
...
+* The [Terraform CLI](https://developer.hashicorp.com/terraform/tutorials/aws-get-started/install-cli) (1.4.6+) must be installed.
+* An [AWS account](https://aws.amazon.com/free/?all-free-tier) and the [associated credentials](https://docs.aws.amazon.com/IAM/latest/UserGuide/security-creds.html) that allow you to create resources. These credentials must be configured for the AWS provider (see [Authentication and Configuration](https://registry.terraform.io/providers/hashicorp/aws/latest/docs#authentication-and-configuration) section in AWS terraform provider documentation.)
+* The [ROSA getting started AWS prerequisites](https://console.redhat.com/openshift/create/rosa/getstarted) must be completed.
+* A valid [OpenShift Cluster Manager API Token](https://console.redhat.com/openshift/token) must be configured (see [Authentication and configuration](https://registry.terraform.io/providers/terraform-redhat/rhcs/latest/docs#authentication-and-configuration) for more information.)
 
-Installation of the following CLI tools is recommended:
+We recommend the installation of the following CLI tools:
```
Comment (EricPonvelle) [inline README.md:32]: s/We recommend the installation of the following CLI tools/We recommend you install the following...

I thought this version read better.

File: modules/machine-pool/README.md [hunk 3]
```diff
@@ -2,17 +2,13 @@
...
+This Terraform sub-module manages the machine pool for ROSA Classic clusters. It enables you to efficiently configure and scale machine pools after cluster deployment, ensuring optimal resource allocation and performance for workloads within the ROSA Classic cluster environment. With this module, you can easily adjust the size and specifications of machine pools, facilitating seamless adaptation to changing workload demands and operational requirements in ROSA Classic clusters.
 
-For more info see [Using machine pools on your cluster](https://registry.terraform.io/providers/terraform-redhat/rhcs/latest/docs/guides/machine-pool)
+For more information, see [Using machine pools on your cluster](https://registry.terraform.io/providers/terraform-redhat/rhcs/latest/docs/guides/machine-pool).
 
 ## Prerequisites
 
-Ensure that you have an existing Red Hat OpenShift Service on AWS (ROSA) cluster deployed on AWS. (see [rosa-cluster-classic sub-module](../rosa-cluster-classic/README.md))
+Ensure that you have an existing Red Hat OpenShift Service on AWS (ROSA) Classic cluster. (see [rosa-cluster-classic sub-module](../rosa-cluster-classic/README.md))
```
Comment (EricPonvelle) [inline modules/machine-pool/README.md:11]: s/see/See

File: modules/operator-policies/README.md [hunk 4]
```diff
@@ -2,19 +2,19 @@
...
-Included in this sub-module are the following permissions:
+The following permissions are included in this sub-module:
 - ROSA Ingress Operator IAM policy: This IAM policy grants the ROSA Ingress Operator the necessary permissions to oversee external access to a cluster.
 - ROSA back-end storage IAM policy: This IAM policy is essential for ROSA to manage back-end storage through the Container Storage Interface (CSI).
 - ROSA Machine Config Operator policy: This IAM policy furnishes the ROSA Machine Config Operator with the permissions required to execute core cluster functionalities.
 - ROSA Cloud Credential Operator policy: This IAM policy offers the ROSA Cloud Credential Operator the necessary permissions for managing cloud provider credentials.
 - ROSA Image Registry Operator policy: This IAM policy provides the ROSA Image Registry Operator with permissions to manage the OpenShift image registry storage in AWS S3 for a cluster.
 
-For more info see:
+For more information, see:
```
Comment (EricPonvelle) [inline modules/operator-policies/README.md:14]: A bit of work, but could we structure these like our docs in sentence style? I just wanted to keep that consistency.

File: modules/vpc/README.md [hunk 5]
```diff
@@ -2,7 +2,7 @@
...
+This Terraform sub-module facilitates using Bring Your Own VPC (BYO VPC) for ROSA Classic clusters. It enables you to provision and configure all necessary resources within an existing AWS VPC, ensuring compatibility and seamless integration with ROSA deployments. By leveraging this module, you can efficiently set up your own VPC environment, complete with networking components such as subnets, route tables, internet gateways, NAT gateways, and security groups, tailored specifically for ROSA Classic cluster requirements. This flexibility allows for a smooth change if you prefer to use your own VPC infrastructure while leveraging the capabilities of ROSA.
```
Comment (EricPonvelle) [inline modules/vpc/README.md:5]: s/facilitates using/allows you to?

Just thought maybe we could simplify the language a little.

File: modules/operator-roles/README.md [hunk 6]
```diff
@@ -2,19 +2,19 @@
...
-Included in this sub-module are the following permissions:
+The following permissions are included in this sub-module:
 - ROSA Ingress Operator IAM role.
 - ROSA back-end storage IAM role.
 - ROSA Machine Config Operator role.
 - ROSA Cloud Credential Operator role.
 - ROSA Image Registry Operator role.
 
-For more info see:
+For more information, see:
```
Comment (EricPonvelle) [inline modules/operator-roles/README.md:14]: If it isn't a huge ask, could we restructure this as a single sentence with the links?

**Review by EricPonvelle — COMMENTED** [review]
/lgtm

Just a few small nitpicks

**Review by EricPonvelle — COMMENTED** [review]
/lgtm

**Pull request merged**